### ISSUE TITLE
How to unregister file type association

### ISSUE THREAD

**Issue by Kyle-Ye** [OmranK/LeafSyntaxHighlighter#2]
Before this implementation, I did another simple implementation(see https://github.com/vapor/leaf/issues/189).

And it seemed I cannot remove it.

The workaround in README seems not work.

> View all LaunchServices associations: /System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/LaunchServices.framework/Versions/A/Support/lsregister -dump
> Command-F to search for file type, .leaf in this case. Once you find the path name for the project that registered this file type you can unregister it: /System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/LaunchServices.framework/Versions/A/Support/lsregister -u <path to bundle>

Because in 'lsregister -dump' it has been unregistered, but it can still be seen using `mdls -name kMDItemContentTypeTree ./xx.leaf`

`/System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/LaunchServices.framework/Versions/A/Support/lsregister -dump | fzf`
<img width="602" alt="截屏2021-04-13 09 46 42" src="https://user-images.githubusercontent.com/43724855/114484627-2a353400-9c3d-11eb-95a9-96d91c459c98.png">

<img width="314" alt="截屏2021-04-13 09 45 12" src="https://user-images.githubusercontent.com/43724855/114484518-f528e180-9c3c-11eb-8ea7-f20cfc66d267.png">

Any suggestions? Thanks

**Comment by Kyle-Ye** [comment]
How to reproduce:
1. Create a Mac Document app 'TypeTestA', and export a type 'com.xx.typetesta' with extension 'typetest'. Then `touch 1.typetest & mdls -name kMDItemContentTypeTree ./1.typetest`, you'll get 'com.xx.typetesta'
2. Then no matter what I do, it seems I can't remove it(The 'com.xx.typetesta' in `mdls -name kMDItemContentTypeTree ./1.typetest`)

**Comment by OmranK** [comment]
Yeah working with LaunchServices is a pain. When I was Exporting Type Identifier, I was easily unregistering the type using the commands mentioned in the README. I'm not sure if the command removed the metadata but when I would restart Xcode, the .leaf syntax would no longer be highlighted and the icon would be gone. I did this multiple times while trying to get the icon perfect but one time I tried to register a Document Type and then the commands were no longer working. It wouldn't unregister.

I mentioned this in the README and linked the program Onyx that  I used to fully unregister it. I would try a computer restart first as that might clear the LaunchServices database. If it's still there, try to use the Onyx program to rebuild the LaunchServices database and then restart again. Hopefully that works!
